fw/application/src/i18n/language.h

```diff
@@ -15,6 +15,7 @@ typedef enum {
     LANGUAGE_EN_US,
     LANGUAGE_ZH_TW,
     LANGUAGE_ES_ES,
+    LANGUAGE_IT_IT,
```

Comment:
Please put the new languge defination in the end of the language enum.

Reply:
done, even if i think that sooner or later language should be ordered better, I find confusing Europen, Eastern and Western languages all mixed up, usually all Eu language like spanish, franch, german, italian, portughese, are all close together, I understand English like first also mandarin chinese but for all the others... but ok, is just a personal taste ;)

Reply:
No, I think you did not resolve this. 
You should put LANGUAGE_IT_IT after LANGUAGE_FR_FR

Reply:
Sorry, my mistake, please put LANGUAGE_IT_IT after LANGUAGE_PT_PT and before LANGUAGE_COUNT